Description: Fix build with newer gnome-settings-daemon
Author: Gianluca Sforna <giallu@gmail.com>
Origin: Fedora, fixed in version 1.5.1-3, see also http://lists.fedoraproject.org/pipermail/devel/2010-December/147106.html
Forwarded: no
Reviewed-by: Daniel T Chen <crimsun@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/gpointing-device-settings/+bug/831136
Last-Updated: 2011-09-19

Index: gpointing-device-settings-1.5.1/modules/gnome-settings-daemon-plugins/gsd-pointing-device-plugin.c
===================================================================
--- gpointing-device-settings-1.5.1.orig/modules/gnome-settings-daemon-plugins/gsd-pointing-device-plugin.c	2011-09-19 09:15:58.000000000 -0400
+++ gpointing-device-settings-1.5.1/modules/gnome-settings-daemon-plugins/gsd-pointing-device-plugin.c	2011-09-19 09:15:56.000000000 -0400
@@ -158,6 +158,12 @@
     plugin_class->activate = activate;
     plugin_class->deactivate = deactivate;
 }
+
+
+static void
+gsd_mouse_extension_plugin_class_finalize(GsdMouseExtensionPluginClass *klass)
+{
+}
 /*
 vi:ts=4:nowrap:ai:expandtab:sw=4
 */
